AdvancedHero.h

```diff
@@ -12,7 +12,7 @@ class AdvancedHero : public Hero {
     void levelup(int);
     
 public:
-    AdvancedHero(const std::string&,int,const int);
+    AdvancedHero(const std::string&,int,const int,const double);
```

Comment:
Csak egy megjegyzés, hogy bár valóban nem kötelező a deklarációnál nevet adni a paramétereknek, ez is a dokumentáció része. Ha ezt az osztályt használni akarja egy másik fejlesztő, fogalma se lesz, hogy melyik paraméternek mit kell megadnia. A stringről még sejtheti, hogy az a név, de hogy a 2 int micsoda... Majd a dokumentáció pótlási feladatnál erre is figyeljetek.

Reply:
@oliverosz Ezt akkor a következő dokumentálási fázisnál bepótoljuk!